### PR TITLE
Added_conv_label_suggestor

### PR DESCRIPTION
**Pull request by campbell18**

### CODE REVIEW

File: concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java [hunk 1]
```diff
@@ -146,55 +153,66 @@ public void run(ConcordServerConfiguration configuration,
...
+    }
+    catch (InvocationTargetException e)
+    {
+      if (e.getCause() instanceof InvalidConfigPropertiesException)
+      {
+        log.error("Invalid configuration provided for label suggestor", e);
+      }
+      else
+      {
+        log.error("Unknown error ");
```
Comment (LaurenceTews) [inline concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java]: log error anyway?

File: concord-server/server/src/main/java/com/rabidgremlin/concord/resources/PhrasesResource.java [hunk 2]
```diff
@@ -73,31 +74,38 @@ public Response getNextPhraseToLabel(@ApiParam(hidden = true) @Auth Caller calle
...
+
+			phraseToLabel.setPossibleLabels(possibleLabels);
+
+			return Response.ok().entity(phraseToLabel).build();
+		}
+		catch (UnableToGetSuggestionsException e)
+		{
+			log.error("Failed to get suggestions", e);
+			return Response.status(Status.SERVICE_UNAVAILABLE).build();
+		}
```
Comment (LaurenceTews) [inline concord-server/server/src/main/java/com/rabidgremlin/concord/resources/PhrasesResource.java]: Could move this logic to a new class

File: concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java (hunk 1 above)
Comment (campbell18) [reply concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java]: Yeah I wanted a more specific log message...but may as well just have one and print the exception stacktrace

File: concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java [hunk 3]
```diff
@@ -146,55 +153,59 @@ public void run(ConcordServerConfiguration configuration,
...
     SystemLabelStore systemLabelStore = new SystemLabelStore() {
-		
+
 		@Override
 		public List<SystemLabel> getSystemLabels() {
 			LabelsDao dao = jdbi.onDemand(LabelsDao.class);
-			
+
 			List<Label> labels = dao.getLabels();
 			ArrayList<SystemLabel> systemLabels = new ArrayList<SystemLabel>();
```
Comment (LaurenceTews) [inline concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java]: List<SystemLabel>?

Comment (LaurenceTews) [inline concord-server/server/src/main/java/com/rabidgremlin/concord/ConcordServerApplication.java]: could tidy this up to List<SystemLabel> whilst you're here

**Pull request merged**